docs/platform/porting/graphics-and-ui.md

```diff
@@ -434,28 +512,13 @@ There are several backends which can be used as reference when implementing the
 | `libtdm-exynos` | [platform/adaptation/samsung_exynos/libtdm-exynos](https://review.tizen.org/gerrit/gitweb?p=platform%2Fadaptation%2Fsamsung_exynos%2Flibtdm-exynos.git;a=summary) | Backend for a target device which uses the `exynos` chipset using the DRM interface. Has PP and capture capability, using the exynos-specific DRM interface to support PP. |
 | `libtdm-sprd`   | [platform/adaptation/spreadtrum/libtdm-sprd](https://review.tizen.org/gerrit/gitweb?p=platform%2Fadaptation%2Fspreadtrum%2Flibtdm-sprd.git;a=summary) | Backend for a target device which uses the Spreadtrum chipset using the Spreadtrum-specific `ioctl`. Uses the DRM interface to support `vblank`. Has PP capability, but no capture capability. |
 
-### Testing the Porting Result
-
-TDM offers the `tdm-test-server` tool to allow you to easily test the porting result. The `tdm-test-server` tool is included in the `libtdm-tools` package, which can be downloaded from the platform binary's snapshot repository. Make sure that TBM porting is done before using the following commands, because TDM works on top of TBM.
-
-```
-$ systemctl stop display-manager  (stop the display server)
-$ export XDG_RUNTIME_DIR=/run
-$ export TBM_DISPLAY_SERVER=1
-$ tdm-test-server                 (show all options)
-$ tdm-test-server -a              (test all layers)
-$ tdm-test-server -a -v           (test all layers with vblank events)
-```
-
-The following image shows the result of a test performed using the `tdm-test-server -a` command. The fullscreen buffer is set to the PRIMARY layer, and the small buffer is set to the OVERLAY layer.
-
-**Figure: Tdm-test-server results**
+### Testing Porting Result
 
-![Tdm-test-server results](media/tdm-test-server-result.png)
+TDM offers `tdm-haltests` that allows you to test and verify the porting result. The `tdm-haltests` tool is included in the `libtdm-haltests` package that can be downloaded from the [platform binary's snapshot repository](https://download.tizen.org/snapshots/tizen/unified/latest/repos/standard/packages/). It depends on `gtest` package and it can be downloaded from the [platform's snapshot repository](https://download.tizen.org/snapshots/tizen/unified/latest/repos/standard/packages/).
```

Comment:
Please add "the" before `gtest` package.

Reply:
I got it.

Reply:
Thank you for updating the changes.